.gitignore

```diff
@@ -1,5 +1,6 @@
 *.gem
 *.rbc
+.idea/
```

Comment:
You could achieve the same thing by adding this to `.git/info/exclude` in your local checkout.

Reply:
👍